src/3045R5_quantities_and_units_library.md

```diff
@@ -5165,6 +5164,172 @@ Having different operators for safe floating-point operations and unsafe integer
 hurt generic programming.
 As such, users should instead use safer representation types.
 
+### Integer overflow
+
+To convert a quantity in a program to different units, we need to multiply or divide by a conversion
+factor.  Sometimes, the result is too big to fit in the type: a problem known as _overflow_.  While
+any numerical type can overflow, we focus on integral types here, because they carry the most risk.
+Even the smallest floating point type in common use, `float`, has a range of $10^{38}$, while the
+diameter of the observable universe measured in atomic diameters is "only" about $10^{37}$![^333]
+
+[^333]: Here, we take the radius of the observable universe as 46.6 billion light years, and the
```

Comment:
Why do you use `^333` here and `^2` below? Did you mean `^1` here?

Reply:
Hah... that was just me being lazy.  :sweat_smile:  I didn't know if we had any footnotes before, but I was pretty sure we wouldn't have 333 of them!  I cleaned up the footnote numbering in the latest commits.